Signal/SignalUBSan.supp

```diff
@@ -7,6 +7,10 @@ bool:YapDatabaseAutoViewTransaction.m
 # SocketRocket
 bool:SRWebSocket.m
 
+# SQLCipher
```

Comment:
Not actually related to iOS12/Xcode10b5, but came upon this while doing some stress testing.